src/contracts/KeyperPayer.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract KeyperPayer is Ownable {
+    using SafeERC20 for IERC20;
+
+    mapping(address => uint) private balances;
+    address[] private keypers;
+    address public sptTokenAddress;
+
+    constructor(address tokenAddress) Ownable(msg.sender) {
+        sptTokenAddress = tokenAddress;
+    }
+
+    function SetKeypers(address[] calldata addresses) public onlyOwner {
```

Comment:
I think the expected/widely accepted case for function name in solidity is camelCase:
setKeypers, getKeypers, balanceOf etc